.gitignore

```diff
@@ -25,3 +25,5 @@ secrets.*.env
 docker/circle/ruby_2.2.2.docker.tgz
 docker/circle/dredd.docker.tgz
 docker-compose.override.yml
+
+coverage
```

Comment:
I think you meant for this .gitignore change to be part of your other pull request.